src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/SignatureInformation.cs

```diff
@@ -3,13 +3,46 @@
 
 namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
 {
+    using System;
+    using System.Globalization;
+    using Microsoft.PowerFx.Intellisense;
+    using SignatureInformationCore = Microsoft.PowerFx.Intellisense.SignatureHelp.SignatureInformation;
+
     /// <summary>
     /// Represents the signature of something callable. A signature
     /// can have a label, like a function-name, a doc-comment, and
     /// a set of parameters.
     /// </summary>
     public class SignatureInformation
     {
+        public SignatureInformation() 
+        { 
+        }
+
+        public SignatureInformation(SignatureInformationCore info)
+        {
+            if (info.Parameters != null)
+            {
+                this.Parameters = Array.ConvertAll(info.Parameters, x => new ParameterInformation(x));
+            }
+
+            this.Label = info.Label;
+            this.Documentation = info.Documentation;
+            if (info.GetDisclaimerMarkdown != null)
+            {
+                // Append disclaimer to create a final markdown string to send to LSP. 
+                MarkdownString disclaimer = info.GetDisclaimerMarkdown();
+                string original = info.Documentation;
+
+                this.Documentation = new MarkdownStringHolder
+                {
+                    Value = original + "\r\n" + disclaimer.Markdown
```

Comment:
I will need to verify if \r\n would always work or not. UFB or more like monaco randomly decides which line feed to use \n or \r\n.

Reply:
use \n\n here

Reply:
updated. And wrapped in a handy static field.